packages/schematics/angular/universal/index.ts

```diff
@@ -171,7 +174,7 @@ function addDependencies(): Rule {
   };
 }
 
-function getTsConfigOutDir(host: Tree, architect: Architect): string {
+function getTsConfigOutDir(host: Tree, architect: experimental.workspace.Architect): string {
```

Comment:
It's odd to have something called `Architect` and `Project` being exported out of the workspace... we should rename these to `ArchitectSchema` and `ProjectSchema`.